crispy_forms/tests/results/bootstrap3/test_layout_objects/bootstrap_modal_with_kwargs.html

```diff
@@ -0,0 +1,24 @@
+<form method="post">
+    <div id="id_test" class="modal fade test-class" tabindex="-1" role="dialog" aria-labelledby="id_test-label">
+        <div class="modal-dialog modal-lg" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title text-center" id="id_title_test">This is my modal</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true" style="float: left">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <div class="form-group" id="div_id_first_name">
+                        <label class="control-label requiredField" for="id_first_name">
+                            first name
+                            <span class="asteriskField">*</span>
+                        </label>
+                        <div class="controls">
+                        <input class="form-control inputtext textInput textinput" id="id_first_name" maxlength="5" name="first_name" required type="text">
+                    </div>
+                </div>
```

Comment:
My intention was that buttons would be added as needed. Only reason I added the close button up top is because bootstrap has that built in so there was no extra code needed. I actually thought about implementing a footer but decided against it because I've personally never been a fan of the literal footer. HTML elements that are needed at the bottom can be added to the bottom of the body using css and/or bootstrap.

I've been using this modal on a personal project and finally decided to actually submit a PR to get it merged. Attached are some examples of how I've been using the modal. Everything in modal was also made using crispy forms.

<img width="1172" alt="Example 1 of modal with custom buttons" src="https://user-images.githubusercontent.com/37155082/150426986-662e0a48-4efb-496a-a9cc-7d0343ec4aa8.png">

<img width="1172" alt="Example 2 of modal with custom buttons" src="https://user-images.githubusercontent.com/37155082/150427825-46c964c1-d9a4-44d1-88da-dee609d0ccfb.png">

Reply:
These look great. 😍 

> ultimately passes the form data to AJAX.

Are you using htmx? That seems to be the latest "hotness" for such things?